manager/base.py

```diff
@@ -205,6 +206,7 @@ def get(cls, **kwargs):
         except InvalidArgumentException as exc:
             return cls.format_exception(str(exc), 400)
         except Exception as exc:  # pylint: disable=broad-except
+            LOGGER.error(format_exc())
```

Comment:
pls use LOGGER.exception() for standartized format

Reply:
Fixed